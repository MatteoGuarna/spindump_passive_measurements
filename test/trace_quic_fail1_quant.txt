A test which attempts to contact the Quant server but it is unreachable
