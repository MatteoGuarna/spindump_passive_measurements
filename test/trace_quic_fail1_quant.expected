[
{ "Event": "new", "Type": "QUIC", "Addrs": ["192.168.203.236","64.99.80.121"], "Session": "09440fca-80adf45068f6d37db6b47d (62329:4433)", "Ts": 1558349060897624, "State": "Starting", "Packets1": 0, "Packets2": 0, "Bytes1": 0, "Bytes2": 0 }
]
